### PR TITLE
[browser] Add a "Preferred colour scheme" combobox to Settings. JB#58394

### PR DESCRIPTION
**Pull request by llewelld**
Adds a "Preferred colour scheme" combobox to the Settings page for controlling the WebEngineSettings.colorScheme property.

### CODE REVIEW

**Comment by llewelld** [comment]
Part of a quiver of four PRs:
1. https://github.com/sailfishos/gecko-dev/pull/149
2. https://github.com/sailfishos/qtmozembed/pull/47
3. https://github.com/sailfishos/sailfish-browser/pull/1007
4. https://github.com/sailfishos/sailfish-components-webview/pull/161

File: apps/browser/qml/pages/SettingsPage.qml [hunk 1]
```diff
@@ -307,6 +307,43 @@ Page {
...
             }
+
+            SectionHeader {
+                //: Section Header for Appearance settings
+                //% "Appearance"
+                text: qsTrId("settings_browser-la-appearance")
+            }
+
+            BrowserComboBox {
+                //% "Preferred colour scheme"
```
Comment (rainemak) [inline apps/browser/qml/pages/SettingsPage.qml]:
```suggestion
                //% "Preferred color scheme"
```
Let's stick with US English

**Review by rainemak — APPROVED** [review]
Didn't test on device but looks reasonable.

File: apps/browser/qml/pages/SettingsPage.qml (hunk 1 above)
Comment (llewelld) [reply apps/browser/qml/pages/SettingsPage.qml]: Thanks; all changed.

**Pull request merged**